01-syntax/01-language-basics/chessBoard.js

```diff
@@ -1 +1,15 @@
-// Your code here
+scale = 8;
```

Comment:
Make sure that you fix all code issues in Inspector or just execute

`npm run lint`  before committing/pushing your changes